### PR TITLE
Issue fix

### PR DESCRIPTION
**Pull request by kavinya**
[VGNWB-120] Fixed the issue

### CODE REVIEW

File: concepts/signals.py [hunk 1]
```diff
@@ -12,24 +12,22 @@
...
-def type_post_save_receiver(sender, **kwargs):
-    """
-    When a :class:`.Type` is saved, attempt to resolve it using one of the
-    registered :class:`.AuthorityManager` classes if the :class:`.Type` is
-    not already :prop:`.resolved`\.
-    """
-    instance = kwargs.get('instance', None)
-    if instance:
-        resolve_concept.delay(sender, instance)
+#@receiver(post_save, sender=Concept)
```
Comment (erickpeirson) [inline concepts/signals.py]: Don't forget to uncomment these signal handlers

**Comment by erickpeirson** [comment]
Hey @kavinya ! Looks like Travis choked -- this is probably because the signal handlers in `concepts/signals.py` need to be uncommented.

Also, it looks like you're adding an enormous number of files! Did you use some kind of IDE or project management tool that is generating these? Definitely don't want this stuff in the repo; you could add them to `.gitignore` if you want to continue using whatever software depends on them.

**Comment by erickpeirson** [comment]
Once you take care of those things, it'll be a bit easier for me to look at your code and make comments.

File: concepts/admin.py [hunk 2]
```diff
@@ -8,17 +11,31 @@ def resolve(modeladmin, request, queryset):
...
+
 class ConceptAdmin(admin.ModelAdmin):
     model = Concept
-    search_fields = ('label',)
-    list_display = ('label', 'resolved', 'concept_state', 'typed',)
-    actions = (resolve,)
-    list_filter=('concept_state', 'typed',)
+    search_fields = (   'label',    )
+    list_display = (    'label', 'resolved','concept_state'  )
+    actions = ( resolve,merge_concepts,    )
```
Comment (erickpeirson) [inline concepts/admin.py]: The whitespace is all messed up here. There should be no whitespace between the parens and the quotes, and there should be a space after each comma.

File: concepts/admin.py [hunk 3]
```diff
@@ -8,17 +11,31 @@ def resolve(modeladmin, request, queryset):
...
+    list_filter=('concept_state',)
```
Comment (erickpeirson) [inline concepts/admin.py]: Why was `typed` removed?

File: concepts/admin.py [hunk 4]
```diff
@@ -8,17 +11,31 @@ def resolve(modeladmin, request, queryset):
...
 
 class TypeAdmin(admin.ModelAdmin):
     model = Type
+    list_display = (    'label', 'resolved'  )
```
Comment (erickpeirson) [inline concepts/admin.py]: Whitespace

File: concepts/admin.py [hunk 5]
```diff
@@ -8,17 +11,31 @@ def resolve(modeladmin, request, queryset):
         authorities.resolve(type(obj), obj)
 resolve.verbose_name = 'resolve selected concepts'
 
+def merge_concepts(self, request, queryset):
+    return render_to_response('admin/merge_concepts.html', {'concepts': queryset})
+    add_tag.short_description = "Merge Concepts"
+
 class ConceptAdmin(admin.ModelAdmin):
```
Comment (erickpeirson) [inline concepts/admin.py:18]: Two blank lines above each class declaration

File: concepts/admin.py [hunk 6]
```diff
@@ -8,17 +11,31 @@ def resolve(modeladmin, request, queryset):
         authorities.resolve(type(obj), obj)
 resolve.verbose_name = 'resolve selected concepts'
 
+def merge_concepts(self, request, queryset):
+    return render_to_response('admin/merge_concepts.html', {'concepts': queryset})
+    add_tag.short_description = "Merge Concepts"
```
Comment (erickpeirson) [inline concepts/admin.py]: What is `add_tag` ? I think you want to set the `verbose_name` attribute of the function, like this (note the lack of indent, and lowercase):

``` python
merge_concepts.verbose_name = 'merge concepts'
```

**Comment by erickpeirson** [comment]
@yyk16293 Thanks for cleaning up the PR. See my line comments, above. If you can clean up those things that I indicated, then this is enough to satisfy VGNWB-120 (add the action). For VGNWB-121, however, we'll need to do more work.

File: concepts/admin.py [hunk 7]
```diff
@@ -8,17 +11,31 @@ def resolve(modeladmin, request, queryset):
...
-    actions = (resolve,)
+    actions = (resolve, merge_concepts,)
     list_filter=('concept_state', 'typed',)
 
 class TypeAdmin(admin.ModelAdmin):
     model = Type
-
     list_display = ('label', 'resolved',)
 
+# def export_selected_objects(modeladmin, request, queryset):
```
Comment (erickpeirson) [inline concepts/admin.py]: What's with all of the commented lines?

Comment (yyk16293) [inline concepts/admin.py]: yes Erick, need to remove those

**Pull request merged**